ragnarson-stylecheck.gemspec

```diff
@@ -1,15 +1,15 @@
 Gem::Specification.new do |s|
   s.name          = "ragnarson-stylecheck"
-  s.version       = "0.8.2"
+  s.version       = "1.0.0"
```

Comment:
why 1.0?

Reply:
Becouse due to change of variable names of configuration file, new build is no longer backward-compatible. Earlier versions of rubocop uses other names, so due to fact that API is no longer compatible I though increasing major version is right approach. 
Please correct me if I'm wrong.

Reply:
I'd say pre-1.0 everything can change the api, so 0.9.x would make sense.

Reply:
I pushed update version information as you recommends :) Should be up now.
Thanks for feedback